src/components/stat/stat--row.twig

```diff
@@ -14,7 +14,8 @@
 				"stat_title": item.stat_title,
 				"stat_prefix": item.stat_prefix,
 				"stat_suffix": item.stat_suffix,
-				"stat_summary": item.stat_summary
+				"stat_summary": item.stat_summary,
+                "stat_content": item.stat_content
```

Comment:
Last item in an object definition should be followed with a comma. Also, the spacing on this seems wrong.